examples/ibm-metrics-router/variables.tf

```diff
@@ -10,7 +10,7 @@ variable "metrics_router_target_name" {
   default     = "my-mr-target"
 }
 variable "metrics_router_target_destination_crn" {
-  description = "The CRN of a destination service instance or resource."
+  description = "The CRN of a destination service instance or resource. Ensure you have a service authorization between IBM Cloud Metrics Routing and your Cloud resource. Read [S2S authorization](https://cloud.ibm.com/docs/metrics-router?topic=metrics-router-target-monitoring&interface=ui#target-monitoring-ui) for details."
```

Comment:
description = "The CRN of a destination sysdig-monitor instance. Ensure you have a service authorization between IBM Cloud Metrics Routing and your sysdig monitor instance. ....

somehow convey it is the sysdig instance crn we want.